internal/auth/auth.go

```diff
@@ -254,7 +257,22 @@ func Verify(ctx context.Context, opt ...Option) (ret VerifyResults) {
 		if v.requestInfo.DisableAuthzFailures {
 			ret.Error = nil
 			// TODO: Decide whether to remove this
-			v.logger.Info("failed authz info for request", "resource", pretty.Sprint(v.res), "user_id", ret.UserId, "action", v.act.String())
+			err := event.WriteObservation(ctx, op,
+				event.WithHeader(
```

Comment:
Eventually this is turned into JSON. However, because this is an inline struct that does not specify JSON tags, the eventual keys will be `Msg`, `UserId`, etc. (https://play.golang.org/p/1OO7GTs2EC9). Unless that is the preferred format for CloudEvents then probably this needs to have JSON tags specified as our general JSON format is lower snake case.